### ISSUE TITLE
Add file "events" root level key to file.json

### ISSUE THREAD

**Issue by DocOtak** [cchdo/cchdo.ucsd.edu#24]
File events are very important and include things such as "submitted", "merged", "online", etc...
It a way is needed to include these specific events.

There are two forms this might take:

An object with date keys:

``` javascript
{
  "events": {
    "2015-06-09T12:00:00-00:00": {
      "event": "merged",
      "note": "some string",
      "name": "some initials or persons name"
    }
  }
 // all the other keys needed for valid file.json
}
```

Or a list with with event objects:

``` javascript
{
  "events": [
      {
        "date": "2015-06-09T12:00:00-00:00",
        "event": "merged",
        "note": "some string",
        "name": "some initials or persons name"
      }
    ]
  }
 // all the other keys needed for valid file.json
}
```

The choice should probably be governed by how postgresql unpacks the objects or array as we will want the "event" timeline to be generated by the database (for speed).

**Comment by DocOtak** [comment]
It looks like the second option (list of event objects) is the way to go. PG9.4 has a [jsonb_populate_recordset](http://www.postgresql.org/docs/9.4/static/functions-json.html) function that appears unpack everything into a table. We could then do normal relational/SQL stuff to the data.

**Comment by DocOtak** [comment]
Implemented with commits 1dc2ceb and 281f114, the website UI and actual administrative tasks of populating the file events remain.
